storage-stackdriver/src/main/java/zipkin2/stackdriver/translation/TraceTranslator.java

```diff
@@ -66,7 +52,7 @@ public TraceTranslator(String projectId) {
         finishTrace(currentTrace, translatedTraces);
 
         currentTrace = Trace.newBuilder();
-        currentTrace.setProjectId(this.projectId);
+        currentTrace.setProjectId(projectId);
         currentTrace.setTraceId(traceId);
```

Comment:
is there any chance a 128bit ID results in < 32 digits of Hex? I.e., do we left pad with 0's like https://github.com/GoogleCloudPlatform/cloud-trace-java/blob/b39dd53117a315f9fea3ff3508dc4805621c0c80/sdk/core/src/main/java/com/google/cloud/trace/core/TraceId.java#L100

Reply:
nope zipkin2.Span's builder normalizes the IDs (via Span.normalizeTraceId). good Q!